runtimes/common/src/constants.rs

```diff
@@ -152,6 +152,28 @@ pub mod attestation {
 	}
 }
 
+pub mod asset_switch {
```

Comment:
This is the assets stuff, nothing to do with switches.

Reply:
renamed in: [6c2d7ac](https://github.com/KILTprotocol/kilt-node/pull/701/commits/6c2d7ac5559c4a0cd42a50ab018966fa9b5a3cc4)